### PR TITLE
Add Minimax chat model

### PR DESCRIPTION
**Pull request by hetaoBackend**
resolve the merging issues for https://github.com/langchain-ai/langchain/pull/6757

### CODE REVIEW

File: libs/langchain/langchain/llms/minimax.py [hunk 1]
```diff
@@ -29,8 +30,7 @@ class _MinimaxEndpointClient(BaseModel):
     api_key: str
     api_url: str
 
-    @root_validator(pre=True)
```
Comment (baskaryan) [inline libs/langchain/langchain/llms/minimax.py:32]: why remove the root_validator decorator?

Comment (hetaoBackend) [reply libs/langchain/langchain/llms/minimax.py:32]: It was my mistake that we I resolved the conflicts, thanks for reminding me, I will fix it.

Comment (hetaoBackend) [reply libs/langchain/langchain/llms/minimax.py:32]: I've fixed the comment, please help with a review again, thanks! @baskaryan

**Comment by baskaryan** [comment]
lgtm, thank you @hetaoBackend!

**Comment by hetaoBackend** [comment]
> lgtm, thank you @hetaoBackend!

You're very welcome. I am glad to contribute to Langchain, hope to contribute more down the road. 😄

**Pull request merged**